### ISSUE TITLE
[BUG] Wasn't there a warning for games that use Denuvo?

### ISSUE THREAD

**Issue by fantasygone** [IsThereAnyDeal/AugmentedSteam#1948]
### Expected Behavior
I recall there being a red message saying if the game had the drm Denuvo in it but it's not the case anymore

### Actual Behavior
No Warning

![image](https://github.com/IsThereAnyDeal/AugmentedSteam/assets/104144029/06dc3d6b-c104-4947-ae95-cfe75cba3993)
![image](https://github.com/IsThereAnyDeal/AugmentedSteam/assets/104144029/be50ae74-067c-499c-981d-cb6c7e15e534)

### Steps to Reproduce the Problem

  1.  Open Steam page of games with Denuvo (EA/SEGA games)
  2.  Notice the warning is missing

### Specifications

  - Augmented Steam Version: 4.0.1
  - Browser Version: Opera 111.0.5168.43

**Comment by fantasygone** [comment]
Hello @candela97 

Do you think it's possible to put the same warning to games that use Denuvo but don't show it in the second screenshot like Persona 5?

Example, Code Vein has Denuvo but the page doesn't say so. It is mentioned in the game's EULA if you can somehow check that.

Steam: https://store.steampowered.com/app/678960/CODE_VEIN/

EULA: https://store.steampowered.com/eula/678960_eula_0

Basically it uses the game ID so probably doable? Thank you

**Comment by candela97** [comment]
Yes, that's doable, but can you open a new issue so we can assess whether it's worth it?

**Comment by fantasygone** [comment]
> Yes, that's doable, but can you open a new issue so we can assess whether it's worth it?

Done! I would become more reliant to the extension if this was added

**Comment by xeropresence** [comment]
I'm still not seeing the DRM warnings, should there be a new issue?
